examples/seismic/acoustic/operators.py

```diff
@@ -1,9 +1,62 @@
-from sympy import Eq
+from sympy import Eq, solve, Symbol
 
 from devito import Operator, Forward, Backward, DenseData, TimeData, time, t
+from devito.logger import error
 from examples.seismic import PointSource, Receiver
 
 
+def laplacian(field, time_order, m, s):
+    """
+    Spacial discretization for the isotropic acoustic wave equation. For a 4th
+    order in time formulation, the 4th order time derivative is replaced by a
+    double laplacian:
+    H = (laplacian + s**2/12 laplacian(1/m*laplacian))
+    :param field:  Symbolic TimeData object, solution to be computed
+    :param time_order: time order
+    :param m: square slowness
+    :param s: symbol of for the time-step
+    :return: H
+    """
+    if time_order == 2:
+        biharmonic = 0
+    elif time_order == 4:
+        biharmonic = field.laplace2(1/m)
+    else:
```

Comment:
Perhaps we should capture such an error much earlier

Reply:
Not sure, one may to to create new operators with higher order, this is here to ensure that if you use ours, these two orders are the only accepted ones.